Python-packages/covidcast-py/covidcast/geography.py

```diff
@@ -4,14 +4,19 @@
 from typing import Union, Iterable
 
 import pandas as pd
-import pkg_resources
-
-COUNTY_CENSUS = pd.read_csv(
-    pkg_resources.resource_filename(__name__, "geo_mappings/county_census.csv"), dtype=str)
-MSA_CENSUS = pd.read_csv(
-    pkg_resources.resource_filename(__name__, "geo_mappings/msa_census.csv"), dtype=str)
-STATE_CENSUS = pd.read_csv(
-    pkg_resources.resource_filename(__name__, "geo_mappings/state_census.csv"), dtype=str)
+import importlib_resources
+
+county_census = importlib_resources.files(__name__) / 'geo_mappings' / 'county_census.csv'
```

Comment:
since this is the top-level namespace for this sub-package, we should either prefix this variable name with an underscore or avoid creating the variable in the first place.  this is especially true because its name is the lowercase version of the name of a constant, so maybe make the name more descriptive if you want to keep it around.
```suggestion
_county_census_file_path = importlib_resources.files(__name__) / 'geo_mappings' / 'county_census.csv'
```